### DISCUSSION TITLE
Can't select image to insert

### DISCUSSION THREAD

**Discussion by brucebailey** [MasaCMS/MasaCMS#40] [Q&A]
Testing out the new 7.3 on a clean install and it won't allow me to select an image to insert into the content.  Please see video below.  Maybe I am doing something wrong at this point.  Any help or advise would be great. 

https://www.wearemovement.us/masaCantSelectImage.mov

On the backend, I get this error when trying to upload into the wysiwig, but I am able to upload from the file manager.

https://www.wearemovement.us/masaImage2.mov

Is there a log file somewhere I can see the errors?

Server Info:
______________________________

App Server Lucee (5.3.5.92)
DB Server MySQL (5.5.5-10.2.33-MariaDB)
Java Version 1.8.0_192
OS Version Windows Server 2016 (10.0)

**Comment by jochemd** [comment]
Your videos are not visible to users who are not logged in. Considering their T&C give them a blanket permission to share data with advertising companies I am not willing to create an account on Vimeo.

**Reply by brucebailey** [reply]
> Your videos are not visible to users who are not logged in. Considering their T&C give them a blanket permission to share data with advertising companies I am not willing to create an account on Vimeo.

Weird... it was open to the public, but regardless, I uploaded them to a site so it is visible to everyone now.  Thanks for the heads up.

**Reply by JonasEriksson** [reply]
Bruce, the reason your video is requiring a login is that it lacks an age rating, then Vimeo assumes it could be unsafe for kids to watch. Just set an appropriate rating on it and we can all see it w/o logging in :-)

**Reply by brucebailey** [reply]
Got it... I don't use it a lot so assumed "public" was all set.  I see the audience now and will make note for the future, thank you!

**Comment by guustnieuwenhuis** [comment]
Hi Bruce,
We've noticed the same issue with image selection the end of last week. A patch release will follow soon, and will resolve the issue.
